packages/@expo/cli/src/start/server/webpack/WebpackBundlerDevServer.ts

```diff
@@ -165,6 +168,59 @@ export class WebpackBundlerDevServer extends BundlerDevServer {
     }
   }
 
+  async bundleAsync({ mode, clear }: { mode: 'development' | 'production'; clear: boolean }) {
+    // Do this first to fail faster.
+    const webpack = importWebpackFromProject(this.projectRoot);
+
+    if (clear) {
+      await this.clearWebProjectCacheAsync(this.projectRoot, mode);
+    }
+
+    const config = await this.loadConfigAsync({
+      isImageEditingEnabled: true,
+      mode,
+    });
+
+    if (!config.plugins) {
+      config.plugins = [];
+    }
+
+    let bar: ProgressBar | null = null;
+
+    // NOTE(EvanBacon): Prevent breaking in CI.
+    if (process.stderr.clearLine != null) {
```

Comment:
Should we use this in other places too? If so, might be good to move this to something like env/log or something?

Reply:
It's a bit specific to how `progress` works so I moved it inside of `createProgressBar` to ensure all progress bars have a similar behavior.